R/utils.R

```diff
@@ -110,8 +110,25 @@ set_into <- function(x, at, value) {
   x
 }
 
+set_minto <- function(x, atx = NULL, aty = NULL, value) {
```

Comment:
what is this for? seems unused

Reply:
on a style point, I strongly discourage compulsory arguments after optional ones

Reply:
was originally used to set values into matrices, but you're right is unused, will remove.